ResourceStatusSystem/Command/FreeDiskSpaceCommand.py

```diff
@@ -117,7 +117,7 @@ def doMaster( self ):
     for name in elements['Value']:
       diskSpace = self.doNew( name )
       if not diskSpace[ 'OK' ]:
-        gLogger.error( "Unable to calculate free disk space" )
+        gLogger.error( "Unable to calculate free disk space for %s" % name )
```

Comment:
The variable part of the logger error should be the second argument:
   gLogger.error( "Unable to calculate free disk space",  "name: %s" % name )